frontend/device/android/device.lua

```diff
@@ -50,4 +50,25 @@ function Device:init()
     Generic.init(self)
 end
 
+function Device:isFullscreen()
```

Comment:
You still left these in. :-)

Reply:
I don't get it. Can you be more verbose?;)

Reply:
Your use of `local android = require("device/android/device")` was confusing, because elsewhere that always means `local _, android = pcall(require, "android")`.

Anyway, so either you should use Device properly or just stick with the specific Android stuff. Last night I made the judgment call that since this is all very Android-specific, there's probably no use in "littering" Device with it, plus it would be the easier change to make. But that means there's no need to add these methods to this file.